src/test/java/org/llorllale/cactoos/matchers/IsBlankTest.java

```diff
@@ -27,55 +27,53 @@
 
 package org.llorllale.cactoos.matchers;
 
-import org.hamcrest.Description;
-import org.hamcrest.MatcherAssert;
-import org.hamcrest.StringDescription;
-import org.hamcrest.core.IsEqual;
+import org.hamcrest.core.StringContains;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 /**
- * Test case for {@link NotBlank}.
+ * Test case for {@link IsBlank}.
  *
  * @since 1.0.0
  * @checkstyle JavadocMethodCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-public final class NotBlankTest {
+public final class IsBlankTest {
+
+    /**
+     * A rule for handling an exception.
+     */
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
 
     @Test
     public void blank() {
-        MatcherAssert.assertThat(
-            "must not match an empty string",
-            new NotBlank().matchesSafely(
-                "", new Description.NullDescription()
-            ),
-            new IsEqual<>(false)
-        );
+        new Assertion<>(
+            "must match an empty string",
+            () -> "",
+            new IsBlank()
+        ).affirm();
     }
 
     @Test
     public void notBlank() {
-        MatcherAssert.assertThat(
-            "must match a non-empty string",
-            new NotBlank().matchesSafely(
-                "-.$%", new Description.NullDescription()
-            ),
-            new IsEqual<>(true)
-        );
+        this.exception.expect(AssertionError.class);
```

Comment:
@Iprogrammerr, could you please also check that the error message that it explains that the message isn't blank?